docs/concepts/assets/asset-framework.md

```diff
@@ -43,10 +44,12 @@ Asset bridges facilitate using assets from blockchains that aren't Vega. Bridges
 
 When an asset has successfully been proposed and approved through governance, the asset bridge will then need to be updated. Vega validators automatically create a multisig bundle - a collection of signatures indicating their approval of the update. That bundle is then submitted to the bridge before the asset can be deposited and used.
 
+Assets from other chains can also be swapped for assets that are enabled on Vega using Squid router. These assets 
```

Comment:
I'm sure this sentence was going somewhere?

Reply:
Well, I don't remember where so guess I can just delete that haha